transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -221,10 +221,10 @@ def forward(
             if fp8_calibration:
                 # amax of input
                 fp8_meta["scaling_fwd"].amax_history[0][tex.FP8FwdTensors.GEMM1_INPUT] = \
-                    torch.amax(ln_out_total).float()
+                    torch.max(ln_out_total.amax(), -ln_out_total.amin()).float()
```

Comment:
Why not just `torch.max(torch.abs(ln_out_total))'?

Reply:
`aminmax` is better. It's faster than both `torch.max(torch.abs(ln_out_total))` and `torch.max(ln_out_total.amax(), -ln_out_total.amin())`.

But in my test code, `torch.cat(a.aminmax())` raises `RuntimeError: zero-dimensional tensor (at position 0) cannot be concatenated`. So I will use a 2-line code:
```
amin, amax = a.aminmax()
torch.max(-amin, amax)
```